ui/src/app/base/components/Code/Code.js

```diff
@@ -10,21 +10,34 @@ const Code = ({
   numbered,
   ...props
 }) => {
+  const input = useRef(null);
   if (inline) {
     return (
       <code className={className} {...props}>
         {children}
       </code>
     );
   } else if (copyable) {
+    const handleClick = evt => {
+      input.current.focus();
+      input.current.select();
+      try {
+        document.execCommand("copy");
+      } catch (err) {
+        console.error("Copy was unsuccessful");
```

Comment:
Should we maybe use a logging library such as [loglevel](https://github.com/pimterry/loglevel) for example, rather than directly logging to the console?

Reply:
@steverydz structured logging would be nice if we could log report to sentry, but *what* sentry?